api/base/settings/defaults.py

```diff
@@ -326,3 +326,5 @@
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
     },
 }
+
+TEST_MIGRATION = False
```

Comment:
Could we name this, `AUTO_RUN_DATA_MIGRATION` and make sure we have in the `local-dist.py` the `= True` version?